modules/templates/util_misc.go

```diff
@@ -0,0 +1,209 @@
+// Copyright 2023 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package templates
+
+import (
+	"context"
+	"fmt"
+	"html/template"
+	"mime"
+	"path/filepath"
+	"strings"
+	"time"
+
+	activities_model "code.gitea.io/gitea/models/activities"
+	repo_model "code.gitea.io/gitea/models/repo"
+	"code.gitea.io/gitea/modules/git"
+	giturl "code.gitea.io/gitea/modules/git/url"
+	"code.gitea.io/gitea/modules/json"
+	"code.gitea.io/gitea/modules/log"
+	"code.gitea.io/gitea/modules/repository"
+	"code.gitea.io/gitea/modules/svg"
+
+	"github.com/editorconfig/editorconfig-core-go/v2"
+)
+
+func SortArrow(normSort, revSort, urlSort string, isDefault bool) template.HTML {
+	// if needed
+	if len(normSort) == 0 || len(urlSort) == 0 {
+		return ""
+	}
+
+	if len(urlSort) == 0 && isDefault {
```

Comment:
Something is strange here. I think the third bool can be removed?

Reply:
> Move other code (no change) to util_avatar/util_render/util_misc (no need to propose changes for them)

I really do not want to introduce too many untested changes. It's just old code.